hwdb.d/60-keyboard.hwdb

```diff
@@ -1753,7 +1753,7 @@ evdev:input:b0003v258Ap001E*
 
 # Plantronics .Audio 626 DSP
 evdev:input:b0003v047FpC006*
- KEYBOARD_KEY_b002f=f20                                 # Microphone mute button; should be micmute
```

Comment:
KEY_MICMUTE is the default HID mapping for 0xb002f, so this entire entry can be dropped.